src/workflows/airqo_etl_utils/data_sources.py

```
@@ -1,11 +1,16 @@
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)